src/PhenX.EntityFrameworkCore.BulkInsert/Options/OnConflictOptions.cs

```diff
@@ -8,9 +8,12 @@ namespace PhenX.EntityFrameworkCore.BulkInsert.Options;
 public abstract class OnConflictOptions
 {
     /// <summary>
-    /// Optional condition to apply on conflict.
+    /// Optional condition to apply on conflict, in raw SQL.
+    /// The pseudo tables `INSERTED` and `EXCLUDED` can be used to reference data :
+    /// * `INSERTED` refers to the data already in the target table.
+    /// * `EXCLUDED` refers to the new data, being in conflict.
     /// </summary>
-    public string? Condition {get; set; }
+    public string? RawWhere { get; set; }
```

Comment:
I would make this a function, then you do not have to remember the pseudo table names and it is easier to change them if needed.

Reply:
What would be the signature? With two string parameters with the pseudo table names?